wallets/rn_cli_wallet/src/provider/NotifyClientProvider.tsx

```diff
@@ -0,0 +1,208 @@
+import {useEffect, useState} from 'react';
+
+import NotifyClientContext, {
+  NotificationsState,
+  NotifyNotification,
+} from '../context/NotifyClientContext';
+import {NotifyClient, NotifyClientTypes} from '@walletconnect/notify-client';
+import {Alert} from 'react-native';
+import {ENV_PROJECT_ID} from '@env';
+import cloneDeep from 'lodash.clonedeep';
+import {useSnapshot} from 'valtio';
+import SettingsStore from '@/store/SettingsStore';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {SYM_KEY_PREFIX} from '@/constants/Storage';
+
+export const NotifyClientProvider: React.FC<{
+  children: React.ReactNode;
+}> = ({children}) => {
+  const {eip155Address: address} = useSnapshot(SettingsStore.state);
+
+  const [isRegistered, setIsRegistered] = useState(false);
+  const [initializing, setInitializing] = useState(false);
+  const [notifyClient, setNotifyClient] = useState<NotifyClient | undefined>(
+    undefined,
+  );
+  const [account, setAccount] = useState<string | undefined>(undefined);
+  const [subscriptions, setSubscriptions] = useState<any>([]);
+  const [notifications, setNotifications] = useState<NotificationsState>({});
+
+  function handleUpdateSymkeys(
+    _subscriptions: NotifyClientTypes.NotifySubscription[],
+  ) {
+    _subscriptions.map(async ({topic, symKey}) => {
+      const symkey = `${SYM_KEY_PREFIX}${topic}`;
+      const existingSymKey = await AsyncStorage.getItem(symkey);
+
+      if (!existingSymKey) {
+        await AsyncStorage.setItem(symkey, symKey);
+      }
+    });
+  }
+
+  useEffect(() => {
+    if (address) {
+      setAccount(prevAccount => {
+        if (!prevAccount && address) {
+          getActiveSubscriptions();
+        }
+
+        return `eip155:1:${address}`;
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [address]);
+
+  useEffect(() => {
+    if (!notifyClient) {
+      console.log('notify client not initialized');
```

Comment:
Since this is an `useEffect` and expected to have a state which `notifyClient` is not initialized yet, we don't have to log something